action.yml

```diff
@@ -51,6 +51,12 @@ runs:
       with:
         skip-commit-verification: ${{ inputs.skip-commit-verification }}
         skip-verification :  ${{ inputs.skip-verification }}
+    - name: Warn if action skipped
+      uses: actions/github-script@v6
+      if: github.event_name == 'pull_request' && (github.actor != 'dependabot[bot]' || inputs.skip-verification == 'true')
+      with:
+        script: |
+          core.warning(`Expected author 'dependabot[bot]' but received '${context.actor}'. Skipping`)
     - name: Merge/approve PR
       uses: actions/github-script@v6
       if: github.event_name == 'pull_request' && (github.actor == 'dependabot[bot]' || inputs.skip-verification == 'true')
```

Comment:
rather than this solution, shall we not just keep (add) checks in the JS code and keep all the warnings there?

Reply:
I think it can be removed

Reply:
I'm not entirely sure why we did that in the first place but I think it should be fine to remove it and leave it up to the action.

Reply:
I've just removed and it all seems to work fine when testing. I also looked it up and [this](https://github.com/fastify/github-action-merge-dependabot/pull/276/files/3ae29fea7e07144acf343022ab10ee66ce643198#r975431852) was when it was added so I don't think it had a very strong reason, just not running the action at all if not needed to maybe save resources or something like that